roles/sap_vm_provision/defaults/main.yml

```diff
@@ -10,6 +10,8 @@ sap_vm_provision_iac_type: ""
 # aws_ec2_vs , gcp_ce_vm , ibmcloud_vs , ibmcloud_powervs , msazure_vm , ibmpowervm_vm , kubevirt_vm , ovirt_vm , vmware_vm
 sap_vm_provision_iac_platform: ""
 
+# jumphost where ansible playbook will delegate_to
+sap_vm_provision_jumphost: "localhost"
```

Comment:
Change variable name to something that is not jump/jumphost as it is too close to Bastion, which is used by cloud platforms.
Maybe `sap_vm_provision_execution_host`

Reply:
@marcelmamula I've renamed all occurrences of jumphost with execution_host. That way there should be no confusion.